openlibrary/templates/lists/header.html

```diff
@@ -30,7 +30,7 @@ <h1>$:_('<a href="%(href)s">%(name)s</a> / Lists', href=doc.key, name=doc.title)
 
 $def render_user():
     <h1 class="collapse">$:_('<a href="%(href)s">%(name)s</a> / Lists', href=doc.key, name=doc.displayname)</h1>
-    <p class="collapse large brown sansserif">
+    <p class="collapse large brown sansserif" id = list_count>
```

Comment:
```suggestion
    <p class="collapse large brown sansserif" id="list_count">
```
By convention, attribute values are quoted and there are no spaces on either side of the equals sign.